features/org.eclipse.e4.rcp/feature.xml

```diff
@@ -294,6 +294,10 @@
          arch="aarch64"
          version="0.0.0"/>
 
+   <plugin
+         id="org.eclipse.swt.svg"
+         version="0.0.0"/>
+
```

Comment:
In order to include the `org.eclipse.swt.svg` fragment in the provided Eclipse products this part is important.

As far as I can tell no code changes are necessary in Jface to make SVGs usable in Eclipse, aren't they?
Or to ask the other way, round: Would it make sense to just submit this part now?

Reply:
That part is already provided via a separate PR on which this PR is based :-)
- https://github.com/eclipse-platform/eclipse.platform.ui/pull/2593

Reply:
My bad, I confused this PR with the one you mentioned. Added the same comment there:
https://github.com/eclipse-platform/eclipse.platform.ui/pull/2593#discussion_r2013582488